compute/admin_guide/welcome/security_assurance_policy.adoc

```diff
@@ -35,14 +35,27 @@ To report a vulnerability in Prisma Cloud Compute, submit the vulnerability deta
 
 ==== Frequently Asked Questions
 
+* Which Prisma Cloud Compute releases receive security updates?
+
+Prisma Cloud has an 'n-2' support policy that means the current release ('n') and the previous two releases ('n-1' and 'n-2') receive support. Security fixes will be backported only for supported releases. End of Life (EOL) releases will not receive security fixes. 
+For more information, see xref:../welcome/support_lifecycle.adoc[support lifecycle].
+
 *Are security fixes provided for both Prisma Cloud Enterprise and Compute editions?*
 
 Yes, security vulnerabilities are addressed in both the editions.
 
 *Do I have to upgrade my console/defender to get security updates?*
 
 If security fixes are released, you may be required to upgrade either or both the Console and Defender. We recommend that all security releases are adopted immediately. 
-For the full details of which vulnerabilities were fixed in a release, refer to the release notes.
+For the full details of which vulnerabilities were fixed in a release, refer to the 
+
+ifdef::compute_edition[]
```

Comment:
@iansk, will this work for conditionalizing the links to the RN for PCCE and PCEE?

Reply:
@Pubs-MV Yes, that's exactly the syntax you need.